lita-slack.gemspec

```diff
@@ -1,5 +1,5 @@
 Gem::Specification.new do |spec|
-  spec.name          = "lita-slack"
+  spec.name          = "shopify-lita-slack"
   spec.version       = "1.8.0"
   spec.authors       = ["Ken J.", "Jimmy Cuadra"]
   spec.email         = ["kenjij@gmail.com", "jimmy@jimmycuadra.com"]
```

Comment:
I think the right thing to do is drop/change the spec author/email since we are publishing this spec under a different name. Code authorship will be clear via the git history

Reply:
I feel the same way. Okiedokie, I'll fix this up and we'll give it the ol' ⛵️